worker/repository/conf/deployment.yaml

```diff
@@ -129,16 +106,15 @@ databridge.config:
         sslPort: '9711'
         tcpReceiverThreadPoolSize: '100'
         sslReceiverThreadPoolSize: '100'
-        hostName: 172.28.128.4
-
+        hostName: 0.0.0.0
+        host: 172.28.128.4
```

Comment:
@chamindi-a Please recheck this. Are there two parameters? `host` and `hostName`?